### DISCUSSION TITLE
Bank balance

### DISCUSSION THREAD

**Discussion by lux-zero-161** [Paymenter/Paymenter#396] [Ideas]
**Hello! It would be great if you could deposit money into your account**

# Example
Next to your account you will see the money you have added and which you could pay with

I know this is really badly written because I don't speak English very well

**Comment by SirHoDo** [comment]
> **Hello! It would be great if you could deposit money into your account**
> 
> 
> 
> # Example
> 
> Next to your account you will see the money you have added and which you could pay with
> 
> 
> 
> 
> 
> I know this is really badly written because I don't speak English very well

Does this not already exists as credits?

**Reply by lux-zero-161** [reply]
No, I saw it but couldn't find it, it could be that it doesn't even exist
